src/Castle.Facilities.WcfIntegration.Tests/Service/Discovery/LoadBalance/FirstAvailablePolicyTestCase.cs

```diff
@@ -1,107 +0,0 @@
-﻿// Copyright 2004-2012 Castle Project - http://www.castleproject.org/
-// 
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-// 
-//     http://www.apache.org/licenses/LICENSE-2.0
-// 
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-namespace Castle.Facilities.WcfIntegration.Tests
-{
-#if DOTNET40
```

Comment:
These tests probably should stay. The `DOTNET40` symbol used to be defined for both .NET 4.0 and .NET 4.5 builds, so these went into the .NET 4.5 builds.